packages/cody/src/lib/hooks/query-files/fe/queries/__service__/use-__fileName__.ts__tmpl__

```diff
@@ -0,0 +1,26 @@
+import {Api} from "@frontend/api";
+import {AxiosResponse} from "axios";
+import {useQuery} from "@tanstack/react-query";
+import {QueryError} from "@frontend/queries/error/query-error";
+import {<%= className %>} from "@app/shared/queries/<%= serviceNames.fileName %>/<%= fileName %>";
```

Comment:
And here is the EJS template in action: 
1. `<%= className %>` -> `{ ...queryNames }`
2. `<%= serviceNames.fileName %>` -> `{ serviceNames }`
3. `<%= fileName %>` -> `{ ...queryNames }`